oem.php

```diff
@@ -62,25 +62,25 @@
         <img src="images/oem/switchboard-about.png" alt="Switchboard About" />
         <p>By providing an <code>oem.conf</code> file, OEMs can fill in the manufacturer name, product name, model number, and manufacturer URL. An image can also be provided which replaces the generic hardware icon.</p>
         <a href="https://github.com/elementary/switchboard-plug-about/#oem-configuration" target="_blank" rel="noopener" class="read-more">Learn More</a>
 
         <h3 id="third-party-repos">Third-Party Repositories</h3>
-        <p>It is highly discouraged to ship elementary OS with software repositories other than the defaults in elementary OS plus a single repository provided and controlled by the OEM. <strong>Third-party repos effectively give root access and the ability to overwrite any system packages to potentially untrusted third parties.</strong> Even if the party is trustworthy, an OEM’s customer’s security and privacy are now at stake if third parties are compromised, reuse their password on multiple services, etc.</p>
-        <p>Further, if a third-party repository ever becomes unmaintained or unpublished, <strong>it may prevent normal system upgrades.</strong> This could hold back potentially serious security and stability updates from reaching the OEM’s customers.</p>
+        <p>It is highly discouraged to ship elementary OS with software repositories other than the defaults in elementary OS plus a single repository provided and controlled by the OEM. <strong>Third-party repos effectively give root access and the ability to overwrite any system packages to potentially untrusted third parties.</strong> Even if the party is trustworthy, an OEM’s customer’s security and privacy are now at stake if third parties are compromised, or they reuse their password on multiple services, etc.</p>
+        <p>Further, if a third-party repository ever goes missing, becomes unmaintained or unpublished, <strong>it may prevent normal upgrades of the system.</strong> This could hold back potentially serious security and stability fixes from reaching the OEM’s customers.</p>
     </div>
 </div>
 
 <div class="grid">
     <div class="third">
-        <h3>News &amp; Announcements</h3>
-        <p>We share frequent updates on development, major announcements, tips for developers, featured apps, and other new content via our Blog.</p>
+        <h3>News and Announcements</h3>
+        <p>Updates on development, major announcements, tips for developers, featured apps, and other new content is available on our blog.</p>
```

Comment:
Eh, this was intentionally worded to emphasize that we frequently provide and update this information; it's not just "available". Maybe:

```suggestion
        <p>We publish frequent updates on development, major announcements, tips for developers, featured apps, and more on our blog.</p>
```